components/locale_core/src/extensions/transform/mod.rs

```diff
@@ -76,6 +76,16 @@ pub(crate) const TRANSFORM_EXT_STR: &str = "t";
 /// let value: Value = "hybrid".parse().expect("Parsing value failed.");
 /// assert_eq!(loc.extensions.transform.fields.get(&key), Some(&value));
 /// ```
+///
+/// # Ordering
+///
+/// This type deliberately does not implement [`Ord`] or [`PartialOrd`] because there are
+/// multiple possible orderings, and the team did not want to favor one over any other.
+///
+/// Instead, there is a method that returns an ordering: [`Transform::total_cmp`].
+///
+/// String comparison (i.e. [`PartialOrd<&str>`]) can be replicated using [`writeable::cmp_str`].
+///
```

Comment:
Issue: Please don't hide the `# Ordering` section down here below examples. We never put importing things after examples, unless they are themselves examples. Please use the concise language I suggested in another comment.